.github/workflows/main.yml

```diff
@@ -5,12 +5,31 @@ on: [push,pull_request]
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        gemfile: [rails_5_2.gemfile, rails_6_0.gemfile, rails_6_1.gemfile, rails_7_0.gemfile, rails_main.gemfile]
+        ruby_version: [2.5, 2.6, 2.7, 3.0]
+        exclude:
+          - gemfile: rails_main.gemfile
+            ruby_version: 2.5
+          - gemfile: rails_main.gemfile
+            ruby_version: 2.6
+          - gemfile: rails_7_0.gemfile
+            ruby_version: 2.5
+          - gemfile: rails_7_0.gemfile
+            ruby_version: 2.6
+          - gemfile: rails_6_0.gemfile
+            ruby_version: 3.0
+          - gemfile: rails_5_2.gemfile
+            ruby_version: 3.0
```

Comment:
maybe we don't need to exclude things?

Reply:
I tried that but...
<img width="951" alt="Screen Shot 2021-12-30 at 19 42 16" src="https://user-images.githubusercontent.com/50113670/147792671-02a95543-a6a4-44ac-9f8b-f3d47908186a.png">